diff -ru ITKv3/Utilities/vxl/vcl/vcl_compiler.h ITKv3-patched/Utilities/vxl/vcl/vcl_compiler.h
--- ITKv3/Utilities/vxl/vcl/vcl_compiler.h	2015-02-24 10:54:54.000000000 -0500
+++ ITKv3-patched/Utilities/vxl/vcl/vcl_compiler.h	2016-07-07 17:40:03.015651756 -0400
@@ -119,6 +119,8 @@
 #  else
 #   define VCL_GCC_40
 #  endif
+# elif (__GNUC__==5)
+#  define VCL_GCC_50
 # else
 #  error "Dunno about this gcc"
 # endif
Only in ITKv3-patched/Utilities/vxl/vcl: vcl_compiler.h~
diff -ru ITKv3/Utilities/vxl/vcl/vcl_new.h ITKv3-patched/Utilities/vxl/vcl/vcl_new.h
--- ITKv3/Utilities/vxl/vcl/vcl_new.h	2015-02-24 10:54:54.000000000 -0500
+++ ITKv3-patched/Utilities/vxl/vcl/vcl_new.h	2016-07-07 17:42:49.579871279 -0400
@@ -15,7 +15,7 @@
 # include <new.h>
 
 // -------------------- gcc with old library
-#elif defined(VCL_GCC) && !defined(GNU_LIBSTDCXX_V3)
+#elif defined(VCL_GCC) && !defined(GNU_LIBSTDCXX_V3) && !defined(VCL_GCC_50)
 # include <new.h>
 
 // -------------------- old MSVC
Only in ITKv3-patched/Utilities/vxl/vcl: vcl_new.h~
